guides/workspaces.md

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
This is missing an entry in the manifest, so it's currently hidden from the navbar. Is there some other way I can preview this? Going here just returns a 404: https://codercom-qpu8omkyu-codercom.vercel.app/docs/coder/v1.28/guides/workspaces